.travis.yml

```diff
@@ -41,36 +41,40 @@ env:
     # them during CI. CI runs on x86_64 and there is a PostBuild command
     # that attempts to execute the compiled-for-m1 tests to have gtest
     # add the list of tests to the cmake test runner.
+    #
+    # FEATURE_PY is off for arm64 macOS because we'd need a cross-compiled version
```

Comment:
Does anything need to be included in the install package for Python or is the wrapper installed from the Python package manager after KTX-Software is installed?

If the former I'd hate to leave it out of the macOS/arm package. Is a cross-compiling version of Python available? If the latter will the extensions be compiled on first use? When I was using GYP, it came as Python source code. On first use it was compiled and saved.

Reply:
It's the latter. We're publishing a source distribution of the Python package, which means that our package should compile itself when it's installed.

Reply:
I replied earlier in the wrong place. Here is my question again:

Does the source distribution need to be included in our installer packages or will people get it from the python package manager to which, if I'm understanding correctly, that workflow_dispatch job uploads the source?

What does FEATURE_PY=ON do?

Reply:
> Does the source distribution need to be included in our installer packages or will people get it from the python package manager to which, if I'm understanding correctly, that workflow_dispatch job uploads the source?

That’s correct. The package manager “pip” will automatically pull the source distribution from PyPI and compile the Python C bindings.

> What does FEATURE_PY=ON do?

* Build the source distribution
* Compile the bindings for the host architecture
* Build a binary distribution
* Run unit tests

Reply:
What is the purpose of building a binary distribution? We are deployng source to PyPl.

Reply:
When available, binary distributions save time.

Reply:
How are binary distributions distributed?

Reply:
By pip? It's the same package but you can provide both source and binary dists

Reply:
So should we be deploying binary distributions to PyPl too?

Reply:
Sorry "pip" is the executable that uses the "PyPI" website to get the source/binary distributions.

We are uploading both dists to PyPI and people will use pip to install them.

Reply:
Ahh! Which binaries is CI building and uploading. Are there more we should add?

Reply:
It’s basically building the glue C code for the Python bindings. It saves compilation time when installing the Python package. This isn’t a necessity but if this package becomes popular, it might make sense to invest in uploading binaries for all platforms.